exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -91,6 +99,25 @@ def __init__(
             if insecure is not None:
                 insecure = insecure.lower() == "true"
 
+        if (
+            not insecure
+            and environ.get(OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE) is not None
+        ):
+            credentials = _get_credentials(
+                credentials, OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE
```

Comment:
I think there is a bug here. If I set `endpoint=https://` and `insecure=True`. It should consider the scheme in the endpoint and load the certificate, but this doesn't do it.

Reply:
yeah, good catch, I think we have the same problem in Trace, will open a new issue to fix that there too.

Reply:
Actually, there is no but, since the parent exporter deals with the `insecure` argument correctly.